Sage/carbon
# chore(help): remove classic theme - FE-3110

removal of classic styling from component, update of stories to CSF and MDX examples and redundant
documentation has been removed from the component folder.

BREAKING CHANGE: classic styling has been removed from the component

fixes FE-3110

### Proposed behaviour
- Help component should no longer support `classic` theme
- Stories should be in CSF and MDX formats
- Redundant documentation should be deleted

### Current behaviour
- Help component supports `classic` theme
- Stories file is in the storiesOf format
- Documentation complies with our previous standards

### Checklist



### Additional context
N/A

### Testing instructions
- `git checkout remove_classic_theme_help`
- `npm start` to start Storybook
- Navigate to the Help component
- Help component should not have an example for classic theme
- Help components documentation should be an MDX example
